sale_order_invoice_address_propose/models/sale_order.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.onchange('partner_id')
+    def _onchange_partner_id(self):
```

Comment:
@TimLai125 I think we'd better avoid updating the domain in an onchange method as the domain won't be permanent that way (i.e. when you refresh the screen after updating a customer, the domain is gone), otherwise.  Can we just apply domains on these fields in the view?

Reply:
Handled by #7